### DISCUSSION TITLE
[Feature] Add Containing Folder as Metadata

### DISCUSSION THREAD

**Discussion by Peda1996** [immich-app/immich#3437] [Feature Request]
### The feature

I'd like to request the implementation of a feature that automatically includes the containing folder / folder path as metadata for each photo. This feature is also implemented within the Aves Gallery App (within the Albums Tab), and realy helps me to organize my local Gallery on my phone. 

This enhancement would greatly improve the organization and management of photos, enabling users to filter, sort, and search by app more easily, and allowes user to more easily delete unwanted photos of certain apps later on.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #1678
